### PR TITLE
docs(Playground): fix and improve endpoint list

### PR DESCRIPTION
**Pull request by soujiokita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,7 +227,9 @@ The missing piece of hentaifox.com - https://sinkaroid.github.io/jandapress/#api
     - "latest", "popular"
   - Example
     - https://janda.sinkaroid.org/hentaifox/get?book=97527
+    - https://janda.sinkaroid.org/hentaifox/search?key=milf
     - https://janda.sinkaroid.org/hentaifox/search?key=milf&page=2&sort=latest
+    - https://janda.sinkaroid.org/hentaifox/randon
```
Comment (sinkaroid) [inline README.md:232]:
```suggestion
    - https://janda.sinkaroid.org/hentaifox/random
```
hey?!

**Review by sinkaroid — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (soujiokita) [reply README.md]: Sorry I need sleep..

**Comment by soujiokita** [comment]
Docker and Playground fail?

**Comment by sinkaroid** [comment]
You don't have write permission on it

**Review by sinkaroid — APPROVED** [review]

**Pull request merged**